server/backend/api/app/integration/idim_proxy.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from http import HTTPStatus
+
+import requests
+import urllib3
+from api.app.requester import Requester
+from api.app.schemas import IdimProxySearchParamIdir
+from api.config import config
+from requests.adapters import HTTPAdapter
+
+LOGGER = logging.getLogger(__name__)
+
+REQUESTER_ACCOUNT_TYPE_INTERNAL = "Internal"
+
+class IdimProxyService():
+    """
+    The class is used for making requests to search IDIR/BCeID information from IDIM Proxy API.
+    See environment setup (local-dev.env) for idim-proxy TEST api-docs.
+    """
+    TIMEOUT = (5, 20) # Timeout (connect, read) in seconds.
+    RETRY = 3
```

Comment:
I think this warrants discussion. Retrying is problematic as it delays the response to the user. Is this recommended by the IDIM guide?

Reply:
Ha, no, it is not from IDIM guide recommendation. Same thought as above and I was trying to be on the safe side just to add a retry since it is easy to add in Python. However, since you point out it "delays" to the user and our frontend probably didn't take into this consideration, maybe I should remove the retry. Can we discuss if there is a value for it when you have time? Thanks.

Reply:
Discussed and remove this. At this case we don't want to retry for user initiated request.